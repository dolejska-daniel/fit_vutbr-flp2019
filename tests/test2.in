1,2,3
ab
1
3
1,a,2
1,a,3
2,,3
2,a,2
