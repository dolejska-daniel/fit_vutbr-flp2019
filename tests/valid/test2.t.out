0,1,2,3,4
abc
0
3,4
0,a,1
1,b,2
1,c,3
2,a,1
3,c,4
4,c,4
